### ISSUE TITLE
Language

### ISSUE THREAD

**Issue by note89** [devfd/react-native-geocoder#36]
I get the address in my native language on IOS, can i get it in english ?

**Comment by abadikaka** [comment]
yes you can but u need modify it, i open PR for it to get result base on language , hope it accepted and u or the other can use it.. @note89

**Comment by abadikaka** [comment]
@note89 the logic is u set ur local language device as "en" and it will retrieve the result on english

**Comment by nes123** [comment]
is there a solution to this issue ?

**Comment by nes123** [comment]
For android just change line 25 in RNGeocoderModule.java to : 

Geocoder geocoder = new Geocoder(reactContext.getApplicationContext(), Locale.ENGLISH);

**Comment by bdipak** [comment]
@devfd I can see a PR already exist for this change : https://github.com/devfd/react-native-geocoder/pull/87/files

could you please approve it and release a new version ?
